site/gatsby-site/src/components/forms/SubmitForm.js

```diff
@@ -104,14 +104,12 @@ const SubmitForm = () => {
     try {
       const date_submitted = format(new Date(), 'yyyy-MM-dd');
 
-      const description = values.description ? values.description : values.text;
-
       const submission = {
         ...values,
         incident_id: values.incident_id == '' ? 0 : values.incident_id,
         date_submitted,
         date_modified: date_submitted,
-        description: description.substring(0, 200),
+        description: values.description || values.text,
```

Comment:
I think this should be just `values.description`. Otherwise, when promoting to an incident, the `description` field will be huge.